x-pack/plugins/task_manager/server/task_running/task_runner.ts

```diff
@@ -617,6 +622,18 @@ export class TaskManagerRunner implements TaskRunner {
         );
       }
     );
+
+    const { eventLoopBlockMs = 0 } = taskTiming;
+    const taskLabel = `${this.taskType} ${this.instance.task.id}`;
+    if (eventLoopBlockMs > this.eventLoopDelayConfig.warn_on_delay) {
+      this.logger.warn(
+        `event loop blocked for at least ${eventLoopBlockMs} ms while running task ${taskLabel}`,
+        {
+          tags: [taskLabel, 'event-loop-blocked'],
```

Comment:
**question:** could we also add `taskType`? This should allow us to break it down by rule type indirectly I think?

Reply:
So, `taskType` as an additional tag value?  Ya, that could be useful.  It's possible it could be noisy.  Let me give it a go, but guessing it will be fine (and useful!)

Reply:
Yeah, I was thinking just as a `tag 👍  I'm not sure of the downsides, if any.

Reply:
changed in cbffd27d8794f15c350bccb78a6f1db7ef0f9917 to add the taskType as well.

I think we'll have to see how well this works out in practice.  The problem I think is going to be the extra noise of the tasktype tags, but I suspect it won't be too bad.  Already when you look at the distribution of `tags` values, after filtering on `tags:event-loop-blocked`, you have to mentally "ignore" the three that are always there: `kibana`,  `logger`, and `event-loop-blocked` - now we'll have the task types as well.  But presumably would only be a handful.

Reply:
Sounds good, thanks!